agent/src/main/java/nu/studer/teamcity/buildscan/agent/BuildScanServiceMessageInjector.java

```diff
@@ -207,32 +210,35 @@ private File getInitScriptsDir() {
     protected File getGradleUserHome() {
         String gradleUserHomeOverride = System.getProperty("gradle.user.home", System.getenv("GRADLE_USER_HOME"));
         return gradleUserHomeOverride == null
-                ? new File(System.getProperty("user.home", System.getenv("USER_HOME")), ".gradle")
-                : new File(gradleUserHomeOverride);
+            ? new File(System.getProperty("user.home", System.getenv("USER_HOME")), ".gradle")
+            : new File(gradleUserHomeOverride);
     }
 
     private String getMavenInvocationArgs(BuildRunnerContext runner) {
-        List<File> extensionJars = new ArrayList<File>();
-        List<String> sysProps = new ArrayList<String>();
+        List<File> extensionJars = new ArrayList<>();
+        List<String> sysProps = new ArrayList<>();
```

Comment:
`agent` is compiled with a Java 6 toolchain in [CI](https://github.com/etiennestuder/teamcity-build-scan-plugin/blob/main/.github/workflows/build-verification.yml#L31). Unfortunately, the diamond operator can't be used then.

```suggestion
        List<File> extensionJars = new ArrayList<File>();
        List<String> sysProps = new ArrayList<String>();
```

Reply:
Argh, right. Changed in https://github.com/etiennestuder/teamcity-build-scan-plugin/pull/373/commits/90f96034f60ab2301893309222081f61a13f25fc